docs/dev/tools/cli.md

```diff
@@ -1190,6 +1191,24 @@ viam.service.vision.v1.VisionService.GetClassificationsFromCamera
 
 # restart a part of a specified machine
 viam machines part restart --machine=123 --part=456
+
+# Copy a single file to the machine with a new name:
+viam machine part cp --organization "org" --location "location" --machine "m1" --part "m1-main" my_file machine:/home/user/
+
+# Recursively copy a directory to the machine with the same name:
+viam machine part cp --part "m1-main" -r my_dir machine:/home/user/
```

Comment:
If you're using a part name instead of a part ID, per the Slack thread don't you need to pass organization and location? Maybe just make all of these IDs instead of names?